.github/workflows/ci.yml

```
@@ -247,10 +247,10 @@ jobs:
         include:
           - type: "qiskit-superstaq"
             pattern: "*_qss.ipynb qiskit-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
           - type: "cirq-superstaq"
             pattern: "*_css.ipynb cirq-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```

this one shouldn't be necessary - `qiskit-superstaq` is already included in `cirq-superstaq[examples]` (we need it for ibm pulse examples)
```suggestion
            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```

---

ahh sorry this was wrong :/ we still need it to make sure we install the local version

---

Sounds good, fixed in [17a46fd](https://github.com/Infleqtion/client-superstaq/pull/899/commits/17a46fd37ef5a3916ed063147aa2a0280b2606b5).